SocialCareCaseViewerApi.Tests/V1/UseCase/AllocationsUseCaseTests.cs

```diff
@@ -78,7 +78,7 @@ public void ListAllocationsByWorkerEmail()
             var request = new ListAllocationsRequest() { WorkerEmail = "test@example.com" };
             var gatewayResponse = new List<Allocation> { new Allocation() { AllocatedWorker = "Test Worker" } };
 
-            _mockDatabaseGateway.Setup(x => x.SelectAllocations(0, 0, "test@example.com"))
+            _mockDatabaseGateway.Setup(x => x.SelectAllocations(0, 0, "test@example.com", 0))
```

Comment:
To ensure the updated use case works as expected would be good to add a test that checks the gateway gets called by correct team_id when provided. I noticed the use case tests are lacking coverage somewhat, but if we improve it for all new features then it would be heading towards better coverage.

Reply:
Done in [9f3bf57](https://github.com/LBHackney-IT/social-care-case-viewer-api/pull/610/commits/9f3bf57c4f2ed74f5302730c840c75520bb00a3a)